container/debian8/builders/bazel/BUILD

```diff
@@ -59,6 +55,18 @@ language_tool_layer(
     ],
 ) for bazel_version, _ in BAZEL_VERSION_SHA256S.items()]
 
+# Set up `entrypoint` and cmd in the container. These cannot be down in the
+# `toolchain_container` rule as they would be consumed by intermediate
+# `container_image` rule in its implementation.
+# TODO(xingao): enable setting things like `entrypoint` and `cmd` in
+# `toolchain_container`.
+[container_image(
+    name = "bazel_%s" % bazel_version,
+    base = ":bazel_%s_intermediate" % bazel_version,
+    entrypoint = ["bazel"],
```

Comment:
we should probably use absolute path to binary (lang_image rules all use full path)